### PR TITLE
Add support for right-alignment

### PR DESCRIPTION
**Pull request by marians**
This PR adds a way to configure a column alignment to be right-aligned.

### CODE REVIEW

**Comment by marians** [comment]
PR https://github.com/giantswarm/gsctl/pull/375 actually makes use of this.

File: README.md [hunk 1]
```diff
@@ -38,7 +38,28 @@ As you can see, you just pass in a list of strings. And the result:
...
+instead of `columnize.SimpleFormat`. Example:
+
+```go
+    config := columnize.DefaultConfig()
+    config.ColumnSpec = []*columnize.ColumnSpecification{
+		&columnize.ColumnSpecification{Alignment: columnize.AlignLeft},
+		&columnize.ColumnSpecification{Alignment: columnize.AlignLeft},
+		&columnize.ColumnSpecification{Alignment: columnize.AlignRight},
+	}
+    result := columnize.Format(output, config)
```
Comment (xh3b4sd) [inline README.md:54]: The indention is a bit funny in the code example.

File: columnize_test.go [hunk 2]
```diff
@@ -337,6 +338,47 @@ func TestGetStringFormat02(t *testing.T) {
 	}
 }
 
+func TestTextAlignment(t *testing.T) {
```
Comment (xh3b4sd) [inline columnize_test.go:341]: I think it makes sense to write table driven tests as defined in https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md.

**Review by xh3b4sd — APPROVED** [review]
The code makes me highly uncomfortable but the unit tests create some kind of confidence. This is good. I am not blocking here. Only suggesting a couple of things.

File: columnize_test.go (hunk 2 above)
Comment (marians) [reply columnize_test.go:341]: Since all but ~4 of these tests where provided by the upstream repo we forked from, I'd rather keep it simple.

**Comment by marians** [comment]
Thanks for the review!

**Comment by marians** [comment]
FYI: fixed white space in README separately

**Pull request merged**